### ISSUE TITLE
Rectangle button on Android 6.0 - API 23

### ISSUE THREAD

**Issue by fcin** [wilsonvargas/ButtonCirclePlugin#38]
Button becomes a rectangle instead of square when run on my Samsung A5 Android 6.0 - API 23. It displays properly on android emulator, but when I plug it and debug it on my phone it is a rectangle.

This is the code I use:

```
    <ContentPage.Content>
        <StackLayout>
            <Grid VerticalOptions="CenterAndExpand">
                <test:CircleButton FontIcon="Material"
                                   Icon="md-play-arrow" 
                                   FontSize="30" TextColor="White" 
                                   HeightRequest="70" WidthRequest="70" 
                                   BorderThickness="0" BorderColor="Black" 
                                   BackgroundColor="#455A64" Command="{Binding PlayButtonClickedCommand}"></test:CircleButton>
            </Grid>
        </StackLayout>
    </ContentPage.Content>
```
And here is what it looks like

![screenshot_2018-07-29-12-46-28](https://user-images.githubusercontent.com/16255350/43365470-b319736a-932d-11e8-8b67-9cfe16cd218b.png)

**Comment by wilsonvargas** [comment]
This issue is similiar to #23 it's solved by assigning a style like "**NoShadow**" in your `style.xml` file:

```xml
<style name="NoShadowButton" parent="android:style/Widget.Button">
        <item name="android:stateListAnimator">@null</item>
</style>
```

And then set this style to `android:buttonStyle` in your main style, like this:
```xml
<item name="android:buttonStyle">@style/NoShadowButton</item>
```

For more information see [this sample file](https://github.com/wilsonvargas/ButtonCirclePlugin/blob/master/test/TestAppButtonCircle/TestAppButtonCircle.Android/Resources/values/styles.xml)

**Comment by wilsonvargas** [comment]
This issue was solved in #1 pls check it.
